subsys/suit/execution_mode/src/suit_execution_mode.c

```diff
@@ -33,10 +33,14 @@ void suit_execution_mode_startup_failed(void)
 	case EXECUTION_MODE_FAIL_INSTALL_NORDIC_TOP:
 	/* SUIT processes update candiadate. */
 	case EXECUTION_MODE_INSTALL:
+	/* SUIT processes recovery as foreground update. */
+	case EXECUTION_MODE_INSTALL_FDFU:
```

Comment:
I know `FDFU` is shorter, but it makes the code harder to read.. why not `FOREGROUND_DFU` ? It doesn't seem to long